bridges/snowbridge/pallets/outbound-queue-v2/src/send_message_impl.rs

```diff
@@ -39,7 +42,10 @@ where
 	fn deliver(ticket: Self::Ticket) -> Result<H256, SendError> {
 		let origin = AggregateMessageOrigin::SnowbridgeV2(ticket.origin);
 
-		if ticket.origin != primary_governance_origin() {
+		let governance_origin =
+			AgentIdOf::convert_location(&Here.into()).ok_or(SendError::InvalidOrigin)?;
+
+		if ticket.origin != governance_origin {
 			ensure!(!Self::operating_mode().is_halted(), SendError::Halted);
 		}
```

Comment:
This is all obsolete code, since the pallet will no longer have an operating mode. Please remove.

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/6e1b0142618c433aece64e2798938874cead0b5c